topo_processor/util/conversions.py

```diff
@@ -0,0 +1,13 @@
+def convert_value(value):
```

Comment:
Let's be more intentional for each key/value that we are handling - this method will cause issues where something looks like an integer but should actually be stored as a string.